packages/xod-client-browser/src/containers/App.jsx

```diff
@@ -181,6 +217,7 @@ class App extends client.App {
           onClick(items.renameProject, this.props.actions.requestRenameProject),
           items.separator,
           importProject,
+          onClick(items.openTutorialProject, this.onOpenTutorial),
```

Comment:
I’m sorry, now I think duplicating the item is not the best idea. I think it would be better if tutorial project access will be in the Help menu only.